### ISSUE TITLE
Create Document: The panel closes after creating if there is something selected on the main list. 

### ISSUE THREAD

**Issue by LBoyle** [nittygritty-uk/Atvero-Issues#361]
Discuss: 
What should we do in this case? We could de-select anything on the main page, then we wouldn't trigger  DLMS to close the panel. 

Expected: Create file panel stays open after creating a doc. 

Actual: If something is selected on the main list, the panel will close because we're selecting one of the documents, the reducer force closes all panels to open the info panel.

STR: 
Try it with nothing selected, the panel stays open, as expected. 
Select something first, then create a file, and you'll notice the Create panel will close and the Info panel will open.

**Comment by jim80** [comment]
Agreed, It's annoying behaviour at best. In addition, I'd say it was reasonable on closing the create document panel to have it select the document you've just selected...

**Comment by jim80** [comment]
PR's a small change to unselect everything when placeholder list is refreshed - easy to back out of it if it should turn out to be undesired, but I think it's the way it should work...
